content/knowledge/flare_architecure.md

```diff
@@ -0,0 +1,207 @@
+Each blockchain is validated by exactly one Subnet.
+A Subnet can validate arbitrarily many blockchains.
+A node may be a member of arbitrarily many Subnets and must be part of the primary network.
+Before any of the Subnets can become a member of the network, it must stake some AVAX tokens.
+Clients interact with Avalanche through APIs calls to nodes (each chain has its [own API](https://docs.avax.network/build/avalanchego-apis/README)).
+
+AVAX tokens exist on the X-Chain, where they can be traded, on the P-Chain, where they can be provided as a stake when validating the Primary Network, and on the C-Chain, where they can be used in smart contracts or to pay for gas.
+
+#### P-Chain
+
+The Platform Chain is the metadata blockchain on Avalanche and coordinates validators, keeps track of active subnets, and enables the creation of new subnets.
+The Platform Chain implements the Snowman consensus protocol.
+
+#### C-Chain
+
+With the C-Chain API, clients can create smart contracts.
+It is an example of the Ethereum Virtual Machine that is powered by the Avalanche Chain.
+The C-Chain also serves as the default smart contract blockchain of the Avalanche chain.
+C-Chain is also very compatible with the Solidity smart contract as well as Ethereum tooling.
+Therefore, it makes it easy for Ethereum developers to easily port applications into the Avalanche chain.
+
+#### X-Chain
+
+The X-Chain functions as a decentralized network for creating and trading digital assets.
+It is a representation of a real-world resource such as bonds and equity with specific rules that govern their actions.
+Whenever you initiate a transaction on the Avalanche blockchain, you get to pay a fee in AVAX.
+The Exchange Chain is an example of the Avalanche Virtual Machine (AVM).
+Clients can create and trade assets on the X-Chain and other examples of the AVM with the help of the X-Chain API.
+
+## Flare's FBA
+
+TODO: add how Flare hijacks the codebase to run FBA on it.
+
+## State Connector
+
+The State Connector – Flare’s system to observe the state of underlying chains.
+State Connector is one of the most critical protocols on the Flare Network.
+The purpose of a state connector is to connect non-flare assets (like XRP, Dode, LTC, etc.) with the Flare Network (represented by F-Assets) in a trustless manner, or other words, allow the smart contracts on Flare to act on state changes from underlying blockchains.
+The State Connector adopts other blockchains’ consensus mechanisms and validator ecosystems.
+
+### Advantages
+
+A state connector system is a competitive approach for proving the state of an underlying chain to a smart contract, and it has the following advantages:
+
+* Transaction validity references back to an underlying chain's genesis block: Other approaches like the SPV (Simplified Payment Verification) proof does not check the validity of a transaction, because it is considered harmful for proving the state of an underlying chain to another network.
```

Comment:
```suggestion
* Transaction validity references back to an underlying chain's genesis block: Other approaches like the SPV (Simplified Payment Verification) proof do not check the validity of a transaction, because it is considered harmful for proving the state of an underlying chain to another network.
```